tests/cases/compiler/truthinessCallExpressionCoercion1.ts

```diff
@@ -49,6 +49,21 @@ function checksPropertyAccess() {
 
     // ok
     x.foo.bar ? x.foo.bar : undefined;
+
+    var chrome = {
+        platformKeys: {
+            subtleCrypto() {
+                return {
+                    sign() {},
+                    exportKey() { return true }
+                }
+            }
+        }
+    }
+    // ok
+    if (chrome.platformKeys.subtleCrypto().exportKey) {
+        chrome.platformKeys.subtleCrypto().exportKey

```

Comment:
I think I need more coffee; why is this ok?

Reply:
Because it's "used" in the body of the `if`

Reply:
You just need *a* usage in the body. It doesn't actually have to call it.